### ISSUE TITLE
Lighthouse is warning up forever when MS Edge upgraded to latest version 115.0.1901.183 (Official build) (64-bit)

### ISSUE THREAD

**Issue by daotran** [GoogleChrome/lighthouse#15312]
### FAQ

- [X] Yes, my issue is not about [variability](https://github.com/GoogleChrome/lighthouse/blob/main/docs/variability.md) or [throttling](https://github.com/GoogleChrome/lighthouse/blob/main/docs/throttling.md).
- [X] Yes, my issue is not about a specific accessibility audit (file with [axe-core](https://github.com/dequelabs/axe-core) instead).

### URL

https://glitch.com/

### What happened?

 Hi there,

I got this issue when running Lighthouse for performance testing. It happens at any website.
"Lighthouse is warning up forever when MS Edge upgraded to latest version 115.0.1901.183 (Official build) (64-bit)"

How to reproduce:
1. Visit to https://glitch.com/ (for example)
2. Press F12 to open Chrome DevTools
3. Select Lighthouse tab
4. Select Moblile/ Desktop option to generate Lighthouse report
5.  Click Analyze Page Load button
     Observe that Lighthouse is warning up forever... It does not generate the report.

I can run the report if i use the previous version... but now i cannot run when MS Edge upgraded to latest version 115.0.1901.183 (Official build) (64-bit)"
![Lighthouse_is_warning_up_forever](https://github.com/GoogleChrome/lighthouse/assets/7079119/d388f79d-33bc-4e1d-a61c-635dffbd4aa7)

Could anyone help me to check and resolve it soon?
Thank you.

### What did you expect?

Could anyone help me to check and resolve it soon?
Thank you.

### How were you running Lighthouse?

Chrome DevTools

### Lighthouse Version

I cannot generate the report, so i did not see it's version.

### Chrome Version

Version 115.0.5790.110 (Official Build) (64-bit)

**Comment by kenshanta** [comment]
I'm experiencing the same issue. I started testing on a locally ran project. it stopped working after a 1 day session.
using latest version of Brave browser & MacOS.

Eventually i discovered that the tool wasnt working for all other websites aswell.
It was previously mentioned that brave has a `Remote Debugging` toggle under brave/settings/privacy.
I checked and my version of brave does not have that section to disable.
